v8/spnego/http.go

```diff
@@ -48,6 +48,10 @@ type teeReadCloser struct {
 	io.Closer
 }
 
+var (
```

Comment:
It's best practice to avoid the use of global variables. Rather than doing this a better way would be to pass in the function as an argument. However given that we are effectively just testing a simple if statement I don't think the code complexity of the tests here is worth it. I think just updating the if statement as you have is sufficient.